server/database.js

```diff
@@ -147,6 +147,9 @@ class Database {
         await R.exec("PRAGMA cache_size = -12000");
         await R.exec("PRAGMA auto_vacuum = FULL");
 
+        //Avoid error "SQLITE_BUSY: database is locked" by allowing SQLITE to wait up to 5 seconds to do a write
```

Comment:
```suggestion
        // Avoid error "SQLITE_BUSY: database is locked" by allowing SQLITE to wait up to 5 seconds to do a write
```

Reply:
default value is 0, so it will return the error SQLITE_BUSY immediately if this is not set
more details here: https://stackoverflow.com/questions/25319031/what-is-the-default-value-of-sqlite3-busy-timeout